megamek/src/megamek/common/Aero.java

```diff
@@ -350,6 +350,10 @@ public int getWalkMP(boolean gravity, boolean ignoreheat, boolean ignoremodulara
             if (weatherMod != 0) {
                 j = Math.max(j + weatherMod, 0);
             }
+            if(getCrew().getOptions().stringOption(OptionsConstants.MISC_ENV_SPECIALIST).equals(Crew.ENVSPC_WIND)
+                    && game.getPlanetaryConditions().getWeather() == PlanetaryConditions.WI_TORNADO_F13) {
```

Comment:
```suggestion
                    && (game.getPlanetaryConditions().getWeather() == PlanetaryConditions.WI_TORNADO_F13)) {
```

Reply:
What if it's stronger than that?

Reply:
F4 which is Mech only, no other unit type allowed.